docs/gpu.md

```diff
@@ -1,15 +1,15 @@
 # How to run with PyTorch/XLA:GPU
 
-PyTorch/XLA enables PyTorch users to utilize the XLA compiler which supports accelerators including TPU, GPU, and CPU This doc will go over the basic steps to run PyTorch/XLA on a nvidia gpu instance
+PyTorch/XLA enables PyTorch users to utilize the XLA compiler which supports accelerators including TPU, GPU, and CPU. This doc will go over the basic steps to run PyTorch/XLA on a nvidia GPU instances.
 
 ## Create a GPU instance
-Pytorch/XLA currently publish prebuilt docker images and wheels with cuda11.7/8 and python 3.8. We recommend users to create a GPU instance with corresponding config. For a full list of docker images and wheels, please refer to [this doc](https://github.com/pytorch/xla/tree/jackcao/gpu_doc#-available-images-and-wheels).
+To create a GPU VM in Google Compute Engine, follow the [Google Cloud documentation](https://cloud.google.com/compute/docs/gpus/create-vm-with-gpus).
```

Comment:
This should not be exclusive for Google Cloud, can you update the language to be more nutural? Something along the line of
```
You can either use a local machine with GPU attached or a GPU vm on the cloud. For example in Google Cloud you can follow this doc to create the GPU VM.
```

Reply:
just noted that this is from previosu change, but I think we should make that update anyway

Reply:
Good call. Updated it.